### PR TITLE
Makes Suit Storage Unit constructable

### PR DESCRIPTION
**Pull request by BootlegBow**
## About The Pull Request

Makes the Suit Storage Unit constructable. Requires 2 matter bins, 1 micro-manipulator, 1 micro-laser, and 1 scanning module. Available under Industrial Engineering research and printable at the Engineering lathe.

## Why It's Good For The Game

Things which can't be constructed aren't fun, so making something constructable which wasn't constructable before increases fun

## Changelog
:cl:
add: Suit Storage Unit circuit board
/:cl:

### CODE REVIEW

**Comment by BootlegBow** [comment]
Turns out deconstructing the suit storage unit with something in it now just deletes the item from existence so I'm working on fixing that, if someone could add a do not merge tag for now that'd be great

**Comment by ExcessiveUseOfCobblestone** [comment]
These can be used to clear rads so should not be easily produceable.

**Comment by BootlegBow** [comment]
> These can be used to clear rads so should not be easily produceable.

i could make it higher on the tech tree if that would help, but I can see the point

**Comment by Cyprex** [comment]
> These can be used to clear rads so should not be easily produceable.

I don't see how this is an issue, there are already plenty of these on the station.
I dislike how this is a circuitboard, we already have way too many circuitboards. Would be cool if there was some other category that wasn't bloated as heck where it would fit.

**Comment by BootlegBow** [comment]
> > These can be used to clear rads so should not be easily produceable.
> 
> 
> 
> I don't see how this is an issue, there are already plenty of these on the station. Also decontamination showers do the same and these can presumably be built?
> 
> I dislike how this is a circuitboard, we already have way too many circuitboards. Would be cool if there was some other category that wasn't bloated as heck where it would fit.

I do agree with this, but personally don't know enough about coding in byond to do it any other way

**Comment by ExcessiveUseOfCobblestone** [comment]
Sorry I took a bit of a break, if you could just add maybe a mat cost or something that would be fine. Tech tree positioning changes what part in the round can they be accessed, whereas I want them to not be mass producible once available.

**Comment by BootlegBow** [comment]
> Sorry I took a bit of a break, if you could just add maybe a mat cost or something that would be fine. Tech tree positioning changes what part in the round can they be accessed, whereas I want them to not be mass producible once available.

I'll add maybe a sheet of diamond to the required parts and that should be fine since diamond is decently rare

**Comment by Cyprex** [comment]
>whereas I want them to not be mass producible once available.
Why? Emitters can already be mass produced, SSUs don't seem terribly useful beyond their niche radioactive contamination removal option. And honestly do we even have constructible options for contamination removal? Outside of engineering it tends to be pretty hard to do, which is a pain.

**Comment by BootlegBow** [comment]
Sorry about the merge commit, that's just from my incompetence
Anyway i made them require 2 silver now

File: code/game/machinery/suit_storage_unit.dm [hunk 1]
```diff
@@ -512,7 +506,9 @@
 	if(!state_open)
 		if(default_deconstruction_screwdriver(user, "panel", "close", I))
 			return
-	if(default_pry_open(I))
+/obj/machinery/suit_storage_unit/crowbar_act(mob/living/user, obj/item/I)
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/machinery/suit_storage_unit.dm:509]: you lost a return ..() in the above proc

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]

File: code/game/machinery/suit_storage_unit.dm (hunk 1 above)
Comment (ExcessiveUseOfCobblestone) [reply code/game/machinery/suit_storage_unit.dm:509]: harass me on discord when fixed

**Comment by BootlegBow** [comment]
My incompetence has caught up with me and it's just getting more broken at this stage, going to close the PR because I'm probably not going to continue working on this

**Pull request closed without merging**